CorsixTH/Lua/dialogs/fullscreen/drug_casebook.lua

```diff
@@ -173,7 +172,7 @@ function UICasebook:updateIcons()
       if #req.rooms > 0 then
         for _, room_id in ipairs(req.rooms) do
```

Comment:
If `#req.rooms == 0`, the `for` loop will not perform its body, and effectively just jump to the end as well.
As such, this `if` is of very little value, since the same check is also done inside `for`.